content/en/blog/_posts/2020-04-21-contributor-communication-upstream-marketing.md

```diff
@@ -65,7 +65,7 @@ We have hit the accelerator on an **upstream marketing group** under SIG
 Contributor Experience and we want to tackle this challenge straight on. 
 We've learned in other contributor areas that creating roles for 
 contributors is super helpful - onboarding, breaking down work, and
-ownership. [Here's our team charting the course](https://github.com/kubernetes/community/tree/master/communication/marketing-team).
+ownership. [Here's our team charting the course](https://github.com/kubernetes/community/tree/master/communication/contributor-comms).
```

Comment:
In this k/website, we usually don't update blog older then 1 year.

Could you provide some thought about why the SIG-Doc Leads would break the rule for this? 😊

Reply:
I tend to agree, is there a way to manage the migration so we don't break old links? I've commented on the [original issue with a potential solution to this](https://github.com/kubernetes/community/issues/6641#issuecomment-1176506895).